src/requestCertificate.ts

```diff
@@ -82,6 +85,12 @@ async function requestCertificate ({
       value: new PrintableString({ value: peerId })
     })
   )
+  pkcs10.subject.typesAndValues.push(
+    new AttributeTypeAndValue({
+      type: CertFieldsTypes.publicKey,
+      value: new PrintableString({ value: publicKey })
```

Comment:
You're getting pubkey as base64 encoded string. I think its a waste of space to store it a printable string. Should we store this a BitString?